src/dotnet/Users.Contracts/UserExt.cs

```diff
@@ -0,0 +1,18 @@
+namespace ActualChat.Users;
+
+public static class UserExt
+{
+    public static UserStatus GetStatus(this User user)
+    {
+        var claimValue = user.Claims.GetValueOrDefault(UserConstants.Claims.Status);
+
+        return Enum.TryParse(claimValue, out UserStatus status)
+            ? status
+            : throw new ArgumentOutOfRangeException($"Unexpected status claim value '{claimValue}'");
```

Comment:
Not applicable here, but in general, avoid providing a lot of extra info in exceptions. Logging some internal stuff is fine, but e.g. dumping it to exception message is not (might lead to a security issue).

Reply:
this code has gone